content/22.vaccines.md

```diff
@@ -304,22 +304,67 @@ Limited distribution was shared with the UAE [@url:https://www.nytimes.com/inter
+Again, the vaccine was well tolerated and a vaccine efficacy of 90.4% was reported. 
+All moderate to severe cases of COVID-19 occurred in the placebo group, of which in the majority of cases that were sequenced approximately 79% were identified as a VOC or VOI.
+Vaccine efficacy against cases caused by VOC, for which the Alpha variant was mostly responsible (88.6%), was reported to be 92.6% [@doi:10.1101/2021.10.05.21264567]. 
+The conclusion of both trials indicates that the NVX-CoV2373 vaccine is safe, effective against COVID-19, and those who took part in the trials are considered fully vaccinated [@urlhttps://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US:].
+
+Novavax is now preparing to apply for an EUA to the FDA [@urlhttps://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US:].
+It has also been reported that Novavax initiated booster trials in the U.K. [https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#novavax]. 
+However, the Novavax vaccine has yet to be approved in any country to date but is allowed for emergency use in Indonesia [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#novavax] and according to press releases there has been applications for for approval for use submitted to New Zealand, Australia, the U.K., and Canada [@url:https://ir.novavax.com/press-releases] and for emergency use listing with the WHO [@url:https://ir.novavax.com/2021-11-04-Novavax-Files-COVID-19-Vaccine-for-Emergency-Use-Listing-with-World-Health-Organization].
+Indeed, Novavax has also promised to provide up to 1.1 billion doses to COVAX, however, there has been reports of struggle to maintain production capacity and quality, hindering its production targets [@url:https://www.politico.com/news/2021/10/19/novavax-vaccine-rush-process-global-campaign-516298].
+
+The development of a protein subunit vaccine against SARS-CoV-2 has been a remarkable achievement in the short period of time since the beginning of the COVID-19 pandemic in 2019, particularly considering these types of vaccines have not played a major role in previous pandemics.  
+However, protein subunit vaccines do play a role in public health and have contributed to vaccination against hepatitis B since and pertussis since the 1980s [@doi:10.1128/mBio.01339-14; @doi:10.1371/journal.ppat.1003418] and will likely continue to contribute to public health for the foreseeable future due to ongoing research for vaccines against influenza, SARS-CoV-2, Epstein-Barr virus, dengue virus, and human papillomavirus among others [@doi:10.1016/j.micinf.2014.12.006; @doi:https://covid19.trackvaccines.org/vaccines/; @doi:10.1080/21655979.2016.1191707].
```

Comment:
Maybe add this one to it @doi:10.1016/S0168-1656(99)00107-8

For some reason I couldn't add it myself if you wouldn't mind?

Reply:
```suggestion
However, protein subunit vaccines do play a role in public health and have contributed to vaccination against hepatitis B [@doi:10/d4g86c] and pertussis [@doi:10.1128/mBio.01339-14; @doi:10.1371/journal.ppat.1003418] since the 1980s and will likely continue to contribute to public health for the foreseeable future due to ongoing research in vaccines against influenza, SARS-CoV-2, Epstein-Barr virus, dengue virus, and human papillomavirus among others [@doi:10.1016/j.micinf.2014.12.006; @doi:https://covid19.trackvaccines.org/vaccines; @doi:10.1080/21655979.2016.1191707].
```
Awesome! I checked the reference and it seems like hep b was 1986, so reorganized this sentence a little to match!

Reply:
Perfect, thanks a mill!